x-pack/plugins/synthetics/public/legacy_uptime/state/api/snapshot.test.ts

```diff
@@ -60,7 +59,7 @@ describe('snapshot API', () => {
   });
 
   it('throws an error when response is not ok', async () => {
-    mockResponse = new HttpFetchError('There was an error fetching your data.', 'error', {} as any);
+    mockResponse = new Error('There was an error fetching your data.');
```

Comment:
NIT given the test is passing but, should we use `createHttpFetchError` from the mock package here?